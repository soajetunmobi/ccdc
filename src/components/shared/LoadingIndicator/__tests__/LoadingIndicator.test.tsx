import React from 'react'
import { LoadingIndicator } from 'components/shared/LoadingIndicator'
import { render } from '@testing-library/react'

test('it renders correctly', () => {
  const { container } = render(<LoadingIndicator />)
  expect(container).toMatchSnapshot()
})
