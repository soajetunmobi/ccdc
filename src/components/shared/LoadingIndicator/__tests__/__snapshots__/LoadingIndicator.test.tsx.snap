// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div>
  <div
    class="loader"
  />
</div>
`;
